src/Subscriber/LazyloadSubscriber.php

```diff
@@ -144,19 +144,37 @@ public function insertLazyloadScript()
 
         $inline_args = [
             'threshold' => $threshold,
-            'options'   => [
-                'use_native' => 'true',
-            ],
         ];
 
+        /**
+         * Filters the use of native lazyload
+         *
+         * @since 2.3.3
+         * @param bool $use_native True to enable native lazyload usage.
+         */
+        if (apply_filters('rocket_use_native_lazyload', false)) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
+            $inline_args['options'] = [
+                'use_native' => 'true',
+            ];
+        }
+
         if ($this->option_array->get('images') || $this->option_array->get('iframes')) {
-            $inline_args['elements']['loading'] = '[loading=lazy]';
+            // This filter is documented in src/Subscriber/LazyloadSubscriber.php.
+            if (apply_filters('rocket_use_native_lazyload', false)) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
+                $inline_args['elements'] = isset( $inline_args['elements'] ) ? $inline_args['elements'] : [];
+                $inline_args['elements']['loading'] = '[loading=lazy]';
+            }
         }
 
         if ($this->option_array->get('images')) {
+            $inline_args['elements']['image']            = 'img[data-lazy-src]';
```

Comment:
@Tabrisrp You forgot the 2 others.
```suggestion
            $inline_args['elements'] = isset( $inline_args['elements'] ) ? $inline_args['elements'] : [];
            $inline_args['elements']['image']            = 'img[data-lazy-src]';
```